tutorials/algorithms/11_VarQTE.ipynb

```diff
@@ -0,0 +1,911 @@
+            "              1.66533454e-16+5.00000000e-01j],\n",
+            "            dims=(2, 2))\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "In order to use real time McLachlan principle, it's necessary to call `RealMcLachlanPrinciple` class."
+      ],
+      "metadata": {
+        "id": "-ee7tApy4lsp"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from qiskit.algorithms.time_evolvers.variational import RealMcLachlanPrinciple\n",
+        "\n",
+        "var_principle = RealMcLachlanPrinciple()"
+      ],
+      "metadata": {
+        "id": "U8ytm_9g4kkx"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qknpQG8LFRdX"
+      },
+      "source": [
+        "We also set target time as $t=10$, but change the auxiliary operator to be the magnetization operator. Following procedures are similar to `VarQITE`.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "aux_ops = [magnetization]"
+      ],
+      "metadata": {
+        "id": "Yh7vyZr62ubD"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "-o2B3533nqsU"
+      },
+      "outputs": [],
+      "source": [
+        "from qiskit.algorithms import VarQRTE\n",
+        "\n",
+        "time = 10.0\n",
+        "evolution_problem = TimeEvolutionProblem(hamiltonian, time, aux_operators=aux_ops)\n",
+        "var_qrte = VarQRTE(ansatz, init_param_values,var_principle, Estimator())\n",
+        "evolution_result_re = var_qrte.evolve(evolution_problem)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "We can also obtain the exact solution by `SciPyRealEvolver`. We first create the correponding initial state for exact classical method."
+      ],
+      "metadata": {
+        "id": "ByiDyBuia1fN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "yDWL-Zd5oQwH"
+      },
+      "outputs": [],
+      "source": [
+        "init_cir=ansatz.assign_parameters(init_param_values)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "`SciPyRealEvolver` can help us to get the classical exact result."
+      ],
+      "metadata": {
+        "id": "SimuqpjpbHuU"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ac4twWUqonXm"
+      },
+      "outputs": [],
+      "source": [
+        "from qiskit.algorithms import SciPyRealEvolver\n",
+        "\n",
+        "evolution_problem=TimeEvolutionProblem(hamiltonian, time, initial_state=init_cir, aux_operators=aux_ops)\n",
+        "RTEV=SciPyRealEvolver(1001)\n",
```

Comment:
could you change the evolver instance to lower case? (similarly to other variables that are upper-case)